pennylane/ops/channel.py

```diff
@@ -729,6 +729,9 @@ def __init__(self, K_list, wires=None, id=None):
         if not np.allclose(Kraus_sum, np.eye(K_list[0].shape[0])):
             raise ValueError("Only trace preserving channels can be applied.")
 
+    def _flatten(self):
+        return (self.data,), (self.wires, ())
```

Comment:
Why was the this failing with the default behavior?

Reply:
because it accepts one parameter, a list. And then it unpacks that list when calling `super().__init__`, so `self.data` is that n-parameter list converted to a tuple (in other words, `n` different parameters). But `QubitChannel` expects all the parameters as a single parameter.

Reply:
Ah. ok thanks.